docs/DATA_DICTIONARY.md

```
@@ -190,6 +190,7 @@ For table overview, see [TABLE_INVENTORY.md](../TABLE_INVENTORY.md).
 - [qa_game_audit](#qa_game_audit) (616 rows, 13 cols) — **NEW Session 111** (split from fact_game_status)
 - [qa_goal_accuracy](#qa_goal_accuracy) (8 rows, 10 cols)
 - [qa_scorer_comparison](#qa_scorer_comparison) (8 rows, 8 cols)
+- [qa_event_anomalies](#qa_event_anomalies) (0 rows, 10 cols) — **NEW Prompt 104** (event chain anomaly detection)
```

markdownlint-cli2 (0.22.0)

[warning] Link fragments should be valid

(MD051, link-fragments)